### DISCUSSION TITLE
Are you OK and safe Anton?

### DISCUSSION THREAD

**Discussion by ccoVeille** [Antonboom/testifylint#172] [Q&A]
We barely saw you online for the past weeks, your timeline is empty.

@mmorel-35 and I assumed you might be in holidays, but it seems a bit long.

**Comment by Antonboom** [comment]
@ccoVeille @mmorel-35 

Hi!
I appreciate your concern.

Everything is fine, just several parallel personal circumstances came together and threw me off track for a few months.

I plan to get back to work on `testifylint` before the beginning of October 2024

thanks ❤️

**Reply by ccoVeille** [reply]
I'm happy to see a message from you.

Take care of yourself,

See you
